[2025-01-25T00:17:00.640+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-01-25T00:17:00.657+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: extraction_by_hour_dag_v11.extract_and_load_to_mongodb_station_2 manual__2025-01-25T00:16:59.313815+00:00 [queued]>
[2025-01-25T00:17:00.684+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: extraction_by_hour_dag_v11.extract_and_load_to_mongodb_station_2 manual__2025-01-25T00:16:59.313815+00:00 [queued]>
[2025-01-25T00:17:00.684+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 2
[2025-01-25T00:17:00.697+0000] {taskinstance.py:2889} INFO - Executing <Task(PythonOperator): extract_and_load_to_mongodb_station_2> on 2025-01-25 00:16:59.313815+00:00
[2025-01-25T00:17:00.706+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'extraction_by_hour_dag_v11', 'extract_and_load_to_mongodb_station_2', 'manual__2025-01-25T00:16:59.313815+00:00', '--job-id', '27', '--raw', '--subdir', 'DAGS_FOLDER/extract_data_by_hour.py', '--cfg-path', '/tmp/tmppdwnxvmb']
[2025-01-25T00:17:00.709+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=2274) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-01-25T00:17:00.709+0000] {standard_task_runner.py:105} INFO - Job 27: Subtask extract_and_load_to_mongodb_station_2
[2025-01-25T00:17:00.710+0000] {standard_task_runner.py:72} INFO - Started process 2277 to run task
[2025-01-25T00:17:00.756+0000] {task_command.py:467} INFO - Running <TaskInstance: extraction_by_hour_dag_v11.extract_and_load_to_mongodb_station_2 manual__2025-01-25T00:16:59.313815+00:00 [running]> on host 8969ebd0a926
[2025-01-25T00:17:00.877+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='extraction_by_hour_dag_v11' AIRFLOW_CTX_TASK_ID='extract_and_load_to_mongodb_station_2' AIRFLOW_CTX_EXECUTION_DATE='2025-01-25T00:16:59.313815+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-01-25T00:16:59.313815+00:00'
[2025-01-25T00:17:00.879+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-01-25T00:18:21.338+0000] {logging_mixin.py:190} INFO - Erreur lors de l'insertion dans MongoDB : No servers found yet, Timeout: 30s, Topology Description: <TopologyDescription id: 67942db164ff570e5a995fd5, topology_type: Unknown, servers: [<ServerDescription ('172.17.0.2', 27017) server_type: Unknown, rtt: None>]>
[2025-01-25T00:18:21.347+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-01-25T00:18:21.494+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-01-25T00:18:21.495+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=extraction_by_hour_dag_v11, task_id=extract_and_load_to_mongodb_station_2, run_id=manual__2025-01-25T00:16:59.313815+00:00, execution_date=20250125T001659, start_date=20250125T001700, end_date=20250125T001821
[2025-01-25T00:18:21.542+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-01-25T00:18:21.554+0000] {taskinstance.py:3895} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-01-25T00:18:21.558+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
